---------------------------------------
          Unknown local file
---------------------------------------
 - Open file...
OSError("'src/plugins/support/PythonSupport/tests/data/../../../../../../models/tests/unknown' could not be opened.")

---------------------------------------
          Unknown remote file
---------------------------------------
 - Open file...
OSError("'https://unknown' could not be opened (host unknown not found).")

---------------------------------------
           Local CellML file
---------------------------------------
 - Open file...
 - Run file...
 - Retrieve simulation results...
 - Retrieve states...
 - Retrieve states' values...

---------------------------------------
          Remote CellML file
---------------------------------------
 - Open file...
 - Run file...
 - Retrieve simulation results...
 - Retrieve states...
 - Retrieve states' values...

---------------------------------------
           Local SED-ML file
---------------------------------------
 - Open file...
 - Run file...
 - Retrieve simulation results...
 - Retrieve states...
 - Retrieve states' values...

---------------------------------------
          Remote SED-ML file
---------------------------------------
 - Open file...
 - Run file...
 - Retrieve simulation results...
 - Retrieve states...
 - Retrieve states' values...

---------------------------------------
         Local COMBINE archive
---------------------------------------
 - Open file...
 - Run file...
 - Retrieve simulation results...
 - Retrieve states...
 - Retrieve states' values...

---------------------------------------
        Remote COMBINE archive
---------------------------------------
 - Open file...
 - Run file...
 - Retrieve simulation results...
 - Retrieve states...
 - Retrieve states' values...
